### ISSUE TITLE
Think about moving to riverpod

### ISSUE THREAD

**Issue by rolandgeider** [wger-project/flutter#419]
This is more of a discussion, since it'll be a lot of work

**Comment by Dieterbe** [comment]
i wonder if it can be done gradually over time

**Comment by Dieterbe** [comment]
indeed, it can be done gradually over time. this is a great guide: https://riverpod.dev/docs/from_provider/quickstart

**Comment by rolandgeider** [comment]
this has been done in the meantime
